### PR TITLE
Update docker images for Suredbits Wallet

### PR DESCRIPTION
**Pull request by Christewart**

### CODE REVIEW

File: apps/suredbits-wallet/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "3.7"
 
 services:
   web:
-    image: bitcoinscala/wallet-server-ui:1.9.0-19b72728-SNAPSHOT@sha256:e3a4dfd7748a4cfce714beacca869c9116026c03dda9130987e151531d374290
+    image: bitcoinscala/wallet-server-ui:1.9.1@sha256:34f6f57ffd87b200895a528b37b4fb80c0d508af088753a98dc70740c0d7b93a
```
Comment (nevets963) [inline apps/suredbits-wallet/docker-compose.yml:5]: Can you change the digest to: `sha256:3eb479b106811d523c4e0cfde244949f6c76a27c7d1fe59be9b8b51ba2372649`

This looks like the digest only for `amd64`, but the above can be used with both `amd64` and `arm64`

File: apps/suredbits-wallet/docker-compose.yml [hunk 2]
```diff
@@ -26,12 +26,13 @@ services:
     depends_on: 
       - walletserver
   walletserver:
-    image: bitcoinscala/bitcoin-s-server:1.9.0-93-4b964074-SNAPSHOT@sha256:e657deb2e6f65cce2f780680322f1b1212d9707a395dd57bace20617cb417833
+    image: bitcoinscala/bitcoin-s-server:1.9.1-34-3dc70938-SNAPSHOT@sha256:aa491b19b0f2ce6a35e1c7c259e9233a07a1dc49775f994e835dd818f459347b
```
Comment (nevets963) [inline apps/suredbits-wallet/docker-compose.yml:29]: Can you change the digest to: `sha256:1cd82d19059382f740f7b8acbc2d3aaeaf0c1fd7c662bdbc3ef7b97a27ee181f`

**Review by nevets963 — CHANGES_REQUESTED** [review]
Can you also update the version number within `./apps/registry.json` ?

Otherwise this is good to be merged in.

File: apps/suredbits-wallet/docker-compose.yml (hunk 1 above)
Comment (lukechilds) [inline apps/suredbits-wallet/docker-compose.yml:5]:
```suggestion
    image: bitcoinscala/wallet-server-ui:1.9.1@sha256:3eb479b106811d523c4e0cfde244949f6c76a27c7d1fe59be9b8b51ba2372649
```

File: apps/suredbits-wallet/docker-compose.yml (hunk 2 above)
Comment (lukechilds) [inline apps/suredbits-wallet/docker-compose.yml:29]:
```suggestion
    image: bitcoinscala/bitcoin-s-server:1.9.1-34-3dc70938-SNAPSHOT@sha256:1cd82d19059382f740f7b8acbc2d3aaeaf0c1fd7c662bdbc3ef7b97a27ee181f
```

**Pull request merged**